source/shared/cpp/ObjectModel/HostConfig.h

```diff
@@ -12,6 +12,7 @@ struct SpacingDefinition
     unsigned int right = 0;
     unsigned int top = 0;
     unsigned int bottom = 0;
+    SpacingDefinition(int left = 0, int right = 0, int top = 0, int bottom = 0):left(left), right(right), top(top), bottom(bottom){};
```

Comment:
either ways is good, just concerned that if there would be any side effects of adding compiler flag. let's add Ben Vi from islandwood if there is any side effects of adding c++14 flag.

---
In reply to: [122282237](https://github.com/Microsoft/AdaptiveCards/pull/423#discussion_r122282237) [](ancestors = 122282237)